echopype/convert/parse_azfp.py

```diff
@@ -91,26 +91,32 @@ def load_AZFP_xml(self):
         """
 
         xmlmap = fsspec.get_mapper(self.xml_path, **self.storage_options)
-        root = ET.parse(xmlmap.fs.open(xmlmap.root)).getroot()
+        phase_number = None
+        for event, child in ET.iterparse(xmlmap.fs.open(xmlmap.root), events=("start", "end")):
+            if event == "end" and child.tag == "Phases":
+                phase_number = None
+            if event == "start":
+                if len(child.tag) > 3 and not child.tag.startswith("VTX"):
+                    camel_case_tag = camelcase2snakecase(child.tag)
+                else:
+                    camel_case_tag = child.tag
 
-        for child in root.iter():
-            if len(child.tag) > 3 and not child.tag.startswith("VTX"):
-                camel_case_tag = camelcase2snakecase(child.tag)
-            else:
-                camel_case_tag = child.tag
-            if len(child.attrib) > 0:
-                for key, val in child.attrib.items():
-                    self.parameters[camel_case_tag + "_" + camelcase2snakecase(key)].append(val)
+                if len(child.attrib) > 0:
```

Comment:
Parameters created from attributes of phase-dependent elements are not being assigned a phase number string token (eg, "phase1"). This leads to the inconsistency of phase-dependent parameters where the element values are written into separate parameters (eg, `pings_per_burst_phase1 = 1`, `pings_per_burst_phase2 = 1`) but their attributes are written as an array of a single parameter (eg, `pings_per_burst_units = ['pings', 'pings']`). The change I'm suggesting here and in the next suggestion would make the handling of element attributes similar to that of the element values.

But, note that this doesn't seem to have any actual impact on data that are saved to the EchoData object! Apparently none of the phase-dependent attribute values are being used. Still, having a consistent structure for the parsing of element values and attributes into parameters will avoid confusion in the future.

```suggestion
                if phase_number is not None and camel_case_tag != "phase":
                    camel_case_tag += f"_phase{phase_number}"

                if len(child.attrib) > 0:
```